### PR TITLE
NH-4049 - Remove unused EmitUtil members.

### PR DESCRIPTION
**Pull request by ngbrown**
This is [NH-4049](https://nhibernate.jira.com/browse/NH-4049).

A prerequisite for #633

### CODE REVIEW

File: src/NHibernate/Bytecode/EmitUtil.cs [hunk 1]
```diff
@@ -70,29 +13,25 @@ public static void EmitBoxIfNeeded(ILGenerator il, System.Type type)
...
-
-			typeToOpcode[typeof(long)] = OpCodes.Ldind_I8;
-			typeToOpcode[typeof(ulong)] = OpCodes.Ldind_I8;
-
-			typeToOpcode[typeof(float)] = OpCodes.Ldind_R4;
-
-			typeToOpcode[typeof(double)] = OpCodes.Ldind_R8;
+			typeToOpcode = new Dictionary<System.Type, OpCode>(12)
+			{
+				[typeof(bool)] = OpCodes.Ldind_I1,
```
Comment (hazzik) [inline src/NHibernate/Bytecode/EmitUtil.cs]: Please replace with object initializer

Comment (ngbrown) [inline src/NHibernate/Bytecode/EmitUtil.cs]: Isn't that what this is?  This is what ReSharper suggested replacing the Dictionary initialization with.

Comment (hazzik) [inline src/NHibernate/Bytecode/EmitUtil.cs]: No, this is index initializer. We don't use such syntax anywhere yet.

Comment (ngbrown) [inline src/NHibernate/Bytecode/EmitUtil.cs]: Done.

File: src/NHibernate/Bytecode/EmitUtil.cs [hunk 2]
```diff
@@ -1,67 +1,10 @@
...
-using System.Reflection;
 using System.Reflection.Emit;
 using System.Collections.Generic;
-using NHibernate.Linq;
-using NHibernate.Util;
 
 namespace NHibernate.Bytecode
 {
-	public class EmitUtil
+	public static class EmitUtil
```
Comment (hazzik) [inline src/NHibernate/Bytecode/EmitUtil.cs]: btw, can this be made internal?

Comment (ngbrown) [inline src/NHibernate/Bytecode/EmitUtil.cs]: Yes.  It's only used in `NHibernate.Bytecode.Lightweight`.

Comment (ngbrown) [inline src/NHibernate/Bytecode/EmitUtil.cs]: I made it internal.

**Pull request merged**